config/crd/bases/secrets.hashicorp.com_vaultdynamicsecrets.yaml

```diff
@@ -75,10 +75,19 @@ spec:
               namespace:
                 description: Namespace where the secrets engine is mounted in Vault.
                 type: string
+              params:
+                additionalProperties:
+                  type: string
+                description: Params that can be passed when requesting credentials/secrets.
+                  When Params is set the configured RequestHTTPMethod will be ignored.
+                  See RequestHTTPMethod for more details. Please consult https://developer.hashicorp.com/vault/docs/secrets
+                  if you are uncertain about what 'params' should/can be set to.
+                type: object
               path:
-                description: Path in Vault to get the credentials for, and is relative
-                  to Mount. Please consult https://developer.hashicorp.com/vault/docs/secrets
-                  if one is uncertain about what 'path' should be set to.
+                description: uncertain about what 'path' should be set to. Path in
+                  Vault to get the credentials for, and is relative to Mount. Please
+                  consult https://developer.hashicorp.com/vault/docs/secrets if one
+                  is uncertain about what 'path' should be set to.
```

Comment:
```suggestion
                  consult https://developer.hashicorp.com/vault/docs/secrets if you
                  are uncertain about what 'path' should be set to.
```

Reply:
Fixed in 53b3b367ef1a78cb9d978916f545fc943cd081e6